$NetBSD: patch-ab,v 1.2 2013/06/15 21:18:49 pettai Exp $

Look for the configuration files in SYSCONFDIR instead of /etc

--- lib/DNSCheck/Config.pm.orig	2013-04-11 15:53:42.000000000 +0000
+++ lib/DNSCheck/Config.pm
@@ -58,11 +58,11 @@ sub new {
         croak 'Configuration file ' . $arg{'configfile'} . ' not readable';
     }
 
-    my $config = _get_with_path( $arg{'configfile'}, _catfile( $arg{'configdir'}, 'config.yaml' ), '/etc/dnscheck/config.yaml', );
+    my $config = _get_with_path( $arg{'configfile'}, _catfile( $arg{'configdir'}, 'config.yaml' ), '@SYSCONFDIR@/dnscheck/config.yaml', );
 
     my $default_policy = _get_with_path( _catfile( dist_dir( 'DNSCheck' ), 'policy.yaml' ) );
 
-    my $policy = _get_with_path( $arg{'policyfile'}, _catfile( $arg{'policydir'}, 'policy.yaml' ), '/etc/dnscheck/policy.yaml', );
+    my $policy = _get_with_path( $arg{'policyfile'}, _catfile( $arg{'policydir'}, 'policy.yaml' ), '@SYSCONFDIR@/dnscheck/policy.yaml', );
 
     my $locale;
     if ( $arg{'localefile'} ) {
